### PR TITLE
Fix error in the index range math expression in the docstring of MultiMarginLoss

### LINKED ISSUE #84512: Error in the docs (docstring) of torch.nn.MultiMarginLoss
### 📚 The doc issue

In https://pytorch.org/docs/stable/generated/torch.nn.MultiMarginLoss.html, the index range should be

$i \in \left\\{ 0, \\; \cdots , \\; \text{x.size}(0) - 1 \right\\}$

rather than

$x \in \left\\{ 0, \\; \cdots , \\; \text{x.size}(0) - 1 \right\\}$.

### Suggest a potential alternative/fix

Corresponding corrections in the [docstring of `torch.nn.MultiMarginLoss`](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/loss.py#L1348) should be made as

```where :math:`i \in \left\{0, \; \cdots , \; \text{x.size}(0) - 1\right\}```

cc @svekars @holly1238 @albanD @mruberry @jbschlosser @walterddr @kshitij12345 @saketh-are

### PR DESCRIPTION
**Pull request by wenh06**
Fixes #84512

### CODE REVIEW

**Review by lezcano — APPROVED** [review]
That's right! Thank you for the fix! @cpuhrsch could you stamp it and land it? I don't have the rights to do so

**Review by cpuhrsch — APPROVED** [review]
